### PR TITLE
Fix Sphinx python docstring error: text contrib module (#12949)

### LINKED ISSUE #12949: text contrib module docs errors
## Description
Sphinx is throwing errors on the `contrib.text.embedding` and `contrib.text.vocab` modules.

## Error

```
/home/ubuntu/incubator-mxnet/python/mxnet/contrib/text/embedding.py:docstring of mxnet.contrib.text.embedding.GloVe:40: SEVERE: Unexpected section title.

Properties
----------
/home/ubuntu/incubator-mxnet/python/mxnet/contrib/text/embedding.py:docstring of mxnet.contrib.text.embedding.FastText:54: SEVERE: Unexpected section title.

Properties
----------
/home/ubuntu/incubator-mxnet/python/mxnet/contrib/text/embedding.py:docstring of mxnet.contrib.text.embedding.CustomEmbedding:31: SEVERE: Unexpected section title.

Properties
----------
/home/ubuntu/incubator-mxnet/python/mxnet/contrib/text/embedding.py:docstring of mxnet.contrib.text.embedding.CompositeEmbedding:18: SEVERE: Unexpected section title.

Properties
----------
/home/ubuntu/incubator-mxnet/python/mxnet/contrib/text/vocab.py:docstring of mxnet.contrib.text.vocab.Vocabulary:34: SEVERE: Unexpected section title.
```

### PR DESCRIPTION
**Pull request by frankfliu**
## Description ##
1. Fix invalid docstring section header.
2. Remove copy pasted docstrings.
3. Move property docstring to getter method.
Fixes #12949

### Essentials ###
Please feel free to remove inapplicable items for your PR.
- [X] Changes are complete (i.e. I finished coding on this PR)
- [X] All changes have test coverage:
- Unit tests are added for small changes to verify correctness (e.g. adding a new operator)
- Nightly tests are added for complicated/long-running ones (e.g. changing distributed kvstore)
- Build tests will be added for build configuration changes (e.g. adding a new build option with NCCL)
- [X] Code is well-documented: 
- For user-facing API changes, API doc string has been updated. 
- For new C++ functions in header files, their functionalities and arguments are documented. 
- For new examples, README.md is added to explain the what the example does, the source of the dataset, expected performance on test set and reference to the original paper if applicable
- Check the API doc at http://mxnet-ci-doc.s3-accelerate.dualstack.amazonaws.com/PR-$PR_ID/$BUILD_ID/index.html
- [X] To the my best knowledge, examples are either not affected by this change, or have been fixed to be compatible with this change

### CODE REVIEW

File: python/mxnet/contrib/text/embedding.py [hunk 1]
```diff
@@ -610,25 +591,6 @@ class FastText(_TokenEmbedding):
...
-    Properties
-    ----------
-    token_to_idx : dict mapping str to int
-        A dict mapping each token to its index integer.
-    idx_to_token : list of strs
-        A list of indexed tokens where the list indices and the token indices are aligned.
-    unknown_token : hashable object
-        The representation for any unknown token. In other words, any unknown token will be indexed
-        as the same representation.
-    reserved_tokens : list of strs or None
```
Comment (aaronmarkham) [inline python/mxnet/contrib/text/embedding.py]: I don't see this being added back elsewhere...

File: python/mxnet/contrib/text/embedding.py [hunk 2]
```diff
@@ -610,25 +591,6 @@ class FastText(_TokenEmbedding):
...
-        A list of reserved tokens that will always be indexed.
-    vec_len : int
```
Comment (aaronmarkham) [inline python/mxnet/contrib/text/embedding.py]: I don't see this being added back elsewhere...

File: python/mxnet/contrib/text/embedding.py [hunk 3]
```diff
@@ -610,25 +591,6 @@ class FastText(_TokenEmbedding):
...
-        The length of the embedding vector for each token.
-    idx_to_vec : mxnet.ndarray.NDArray
```
Comment (aaronmarkham) [inline python/mxnet/contrib/text/embedding.py]: I don't see this being added back elsewhere...

File: python/mxnet/contrib/text/embedding.py [hunk 4]
```diff
@@ -506,25 +506,6 @@ class GloVe(_TokenEmbedding):
...
-    Properties
-    ----------
-    token_to_idx : dict mapping str to int
-        A dict mapping each token to its index integer.
-    idx_to_token : list of strs
-        A list of indexed tokens where the list indices and the token indices are aligned.
-    unknown_token : hashable object
-        The representation for any unknown token. In other words, any unknown token will be indexed
-        as the same representation.
-    reserved_tokens : list of strs or None
```
Comment (aaronmarkham) [inline python/mxnet/contrib/text/embedding.py]: I don't see this being added back elsewhere...

File: python/mxnet/contrib/text/embedding.py [hunk 5]
```diff
@@ -506,25 +506,6 @@ class GloVe(_TokenEmbedding):
...
-        A list of reserved tokens that will always be indexed.
-    vec_len : int
```
Comment (aaronmarkham) [inline python/mxnet/contrib/text/embedding.py]: I don't see this being added back elsewhere...

File: python/mxnet/contrib/text/embedding.py [hunk 6]
```diff
@@ -506,25 +506,6 @@ class GloVe(_TokenEmbedding):
...
-        The length of the embedding vector for each token.
-    idx_to_vec : mxnet.ndarray.NDArray
```
Comment (aaronmarkham) [inline python/mxnet/contrib/text/embedding.py]: I don't see this being added back elsewhere...

File: python/mxnet/contrib/text/embedding.py (hunk 1 above)
Comment (frankfliu) [reply python/mxnet/contrib/text/embedding.py]: Those doesn't really exists in the class. They are copy pasted from parent class.

**Comment by frankfliu** [comment]
@aaronmarkham The original docstring was a copy paste mistake. They should not be there.

**Comment by aaronmarkham** [comment]
@mxnet-label-bot add [Doc]

**Pull request merged**